### ISSUE TITLE
Use local binary directory instead of root binary directory

### ISSUE THREAD

**Issue by cybardev** [ghadd/password_learner#1]
I was wondering if installing it to `~/.local/bin/` would allow it to be installable normally via `pip`.

I don't see a reason why `/usr/bin/` needs to be used, but feel free to correct me if I'm wrong.

**Comment by ghadd** [comment]
You are correct. I think it may be worth providing an option for the user to explicitly say "hey, I want to install it to local user", say with "-u" argument. 

Thank you for reaching out.

**Comment by cybardev** [comment]
Yes, that makes more sense than my original suggestion. The choice would then be available the user.

Alternatively, is there a way to test whether `pip` is being run with elevated privilege (`sudo`, `su -c`, `doas`, etc.)? That way, if that's the case then `/usr/bin/` (or better yet, `/usr/local/bin/`) can be written to, and if it's run as regular user then install to `~/.local/bin/`.
